src/test/tests/goUtils.test.ts

```diff
@@ -218,4 +231,20 @@ describe('Go Utils Tests', async () => {
             } as vscode.WorkspaceFolder
         ];
     }
+
+    describe('prepareProjectWorkspace util', () => {
+        it('Exclude all kind of files', () => {
+            // Define the source and target directories for testing
+            const sourceDir: string = path.join(tmpDir, 'prepareProjectWorkspace');
+            const targetDir: string = path.join(tmpDir, 'tmpDir');
+
+            
+            GoUtils.prepareProjectWorkspace(sourceDir, targetDir, '', logManager, () => {
+                return;
+            });
+
+
+            assert.isFalse(fs.existsSync(targetDir), 'The target directory should not exist since all files should be excluded');
+        });
+    });
```

Comment:
```suggestion
    it('Prepare project workspace', () => {
        // Define the source and target directories for testing
        const sourceDir: string = path.join(tmpDir, 'prepareProjectWorkspace');
        const targetDir: string = path.join(tmpDir, 'tmpDir');

        GoUtils.prepareProjectWorkspace(sourceDir, targetDir, '', logManager, () => {
            return;
        });

        assert.isFalse(fs.existsSync(targetDir), 'The target directory should not exist since all files should be excluded');
    });
```